src/main/java/school/lemon/changerequest/java/introduction/hw1/Task3.java

```diff
@@ -2,6 +2,6 @@
 
 public class Task3 {
     public static boolean isBetween(int a, int b, int c) {
-        return false;
+        return ((a <= b) && (c >= b)) || ((a >= b) && (c <= b));
```

Comment:
This equation is almost undebugable.

It's always better to have number of `if-else` operators rather than having such long and probably non-trivial equation.